core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/jdbc/JDBCUserStoreManager.java

```diff
@@ -247,6 +273,30 @@ public JDBCUserStoreManager(DataSource ds, RealmConfiguration realmConfig, int t
         if (log.isDebugEnabled()) {
             log.debug("Ended " + System.currentTimeMillis());
         }
+
+        /* Set waiting time to re-establish connection after the specified retry count on failure attempts
+          if specified otherwise set default time.
+         connectionRetryDelay is not defined by default in the JDBC User store, hence defined a new property
+                           and applied the default values defined for ldap user store
```

Comment:
```suggestion
                           and applied the default values defined for ldap user store.
```
Check other places as well